submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,237 @@
+const url = 'https://randomuser.me/api/?results=12';
+const form = document.forms;
+const content = document.querySelector('.articleBlock');
+
+let friendsCopy = [];
+let friends = [];
+let isSearch = false;
+
+async function getResponse(url) {
+    let response;
```

Comment:
You don't need an outside variable for `response`. Do the `.json()` thing right in the `try` block. You have a faulty logic here, you're getting data from a response outside of `try` in any scenario, even if the data isn't there because you ignore the `catch` block. The only reason your program doesn't crash is that if there _is_ an error you just reset your page, but let's not rely on that. If there is no data - don't extract it from the response.